### ISSUE TITLE
loss = nan..what's the problem?

### ISSUE THREAD

**Issue by shiney5213** [peteryuX/arcface-tf2#9]
I am training model with ms1m_dataset and asian seleb dataset
but loss = Non...
Model is not tranied at all.
mode = 'fit'  -> loss = non
mode = 'eager_ft' -> loss = non
mode = 'eager_fit' -> Out Of memory Error
what's the problem?
please help me and thank you...have a nice day

**Comment by peteryuX** [comment]
Hi, @shiney5213. How did you prepare these two datasets?

**Comment by shiney5213** [comment]
Hi @peteryuX 
1. ms1m_align_112 dataset download
2. asian_seleb dataset : download http://trillionpairs.deepglint.com/overview and crob by matcnn(112 * 112)
3. ms1m_align_112 dataset and asian_align_112 put one directory.
4. make tfrecord file by convert_train_binary_tfrecord.py
5. I want to train by train.py....

Previously, a small dataset and ms1m dataset were combined and trained in the same way.
Training was successful.
However, this time, training is not possible. 
I don't know what's the ploblem...

Sorry and thank you for bothering me

**Comment by peteryuX** [comment]
It sounds weird~ From my experience, loss nan generally presents in two situations, **1. input data have some unexpected values**; **2. loss divided with a near zero values,  which might make gradients too large**. You can trace which loss become nan firstly (like regulization l2 norm loss or arcface loss...?) to crash the training. I would try to figure it out when I am free. Please let me know if you find out the problem before then. Thanks!

**Comment by shiney5213** [comment]
thank you  for your answer.
I'll do it myself.
I'll try and ask for help if I have any further questions later
have a nice day
